test/e2e/app-dir/next-script/index.test.ts

```diff
@@ -0,0 +1,32 @@
+import { createNext } from 'e2e-utils'
```

Comment:
Could you move to `nextTestSetup`? You can easily generate via `pnpm new-test`.

Reply:
Thanks for sharing command! It's awesome. I created new test with `pnpm new-test` now.